packages/profile/src/views/Profile.jsx

```diff
@@ -207,6 +256,64 @@ const Profile = () => {
               <Input key={fieldName} fieldName={fieldName} state={state} dispatch={dispatch} />
             )
           })}
+          <div className='mt-8 mb-2 w-min'>
+            <label className='label block mb-1'>
+              <span className='label-text text-gray-700 capitalize'>Avatar</span>
+            </label>
+
+            <div className='mb-2 text-sm text-gray-700'>
+              Upload a JPEG or PNG. Recommended size is 500 x 500.
+            </div>
+
+            <div className={`my-8 grid place-items-center box-border rounded-full h-80 w-80
+              ${state.avatar ? '' : 'border-dashed border-2 border-gray-400'}`}
+            >
+              {state.avatar
+                ? <img
+                    src={state.avatar} alt='avatar'
+                    className='w-80 h-80 object-cover rounded-full'
+                  />
+                : <span className='text-gray-400'>add an image</span>}
+            </div>
+
+            <div className='my-4'>
+              {state.avatar &&
+                <div className='grid grid-cols-2 gap-x-2'>
+                  <label
+                    className='px-6 py-3 w-full bg-kernel-eggplant-mid text-white rounded text-center cursor-pointer'
+                  >
+                    <input
+                      type='file' accept='image/png, image/jpeg'
```

Comment:
we should use image/* so mobile phones allow you to take a picture with the camera

see https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/accept#unique_file_type_specifiers